e2e/wdio/core/tests/link.e2e-spec.ts

```diff
@@ -27,6 +33,28 @@ describe('Link test suite', () => {
             }
         });
 
+        it('should check that first link is standard type', () => {
+            expect(getElementAriaLabel(links)).toBe('Standard');
+        });
+
+        it('should check that second link is emphasized type', () => {
+            expect(getElementAriaLabel(links, 1)).toBe('Emphasized');
+        });
+
+        it('should check that the last link is inverted type', () => {
+            expect(getElementAriaLabel(links, 5)).toBe('Inverted');
+        });
+
+        it('should check link with right arrow', () => {
+            expect(getElementAriaLabel(links, 3)).toBe('Icon right');
+            expect(isElementDisplayed(rightArrowIcon)).toBe(true);
+        });
+
+        it('should check link with right arrow', () => {
```

Comment:
should check link with LEFT arrow?

Reply:
added this case, just forgot to rename iteration